src/utils.ts

```diff
@@ -65,3 +65,11 @@ export const toSQSError = (err, message: string): SQSError => {
 
     return sqsError
 }
+
+export const isFifo = (queueUrl: string): boolean => {
+    if (!queueUrl) {
+        return false
+    }
+    const { length } = queueUrl
+    return queueUrl.substring(length - 5, length) === ".fifo"
```

Comment:
Is this the standard way to determine fifo? Does every fifo queue have to have this suffix?

Reply:
I don't think so but the way we're using it the suffix is everywhere.

Reply:
Seems a bit fragile 😬 But I guess we don't have more info about the queue in the consumer, right? Maybe we should add parameter to the function that creates the consumer and if it is set, respect that, otherwise fallback to this logic

Reply:
it would be better to get the queue attributes but I think that's overkill

Reply:
I don't think it is overkill 🥸 but yeah, let's not overcomplicate for now I guess.

Reply:
overkill for our usage